save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/demo/DemoView.kt

```diff
@@ -111,3 +89,23 @@ private fun demoView(): FC<DemoViewProps> = FC { props ->
         }
     }
 }
+
+/**
+ * [Props] for DemoView
+ */
+external interface DemoViewProps : Props {
```

Comment:
suggestion to add a flag to disable custom-selector

Reply:
Should be done in PR with several run modes (if you are talking about disabling demoMode selector). Probably it is the task that should be done next